config/locales/de.yml

```diff
@@ -320,20 +320,20 @@ de:
     canton: Kanton
     city: Stadt
     region: Region
-  registered_by: SozialberaterIn
+  registered_by: Sozialberater/in
   rejection_type: &id-rejection-types
-    her: von Freiwillige vor / nach Erstgespräch
+    her: von Freiwillige/r vor / nach Erstgespräch
```

Comment:
this whole label is a little confusing to read. Would this maybe better?
vor/nach dem Erstgespräch von Freiwilige/r ?